services/integration-content-repository/app/api/swagger/swagger.json

```diff
@@ -1128,170 +418,166 @@
       }
     },
     "schemas": {
-      "FlowRequest": {
+      "MutableFlow": {
         "type": "object",
         "required": [
-          "name"
+          "name",
+          "graph",
+          "status",
+          "type",
+          "userId",
+          "workspaceId"
         ],
         "properties": {
-          "type": {
-            "type": "string",
-            "example": "flow"
-          },
-          "oihid": {
-            "type": "string",
-            "example": "123456"
-          },
           "name": {
             "type": "string",
-            "example": "WiceToSnazzy"
+            "example": "My Flow"
           },
-          "status": {
+          "description": {
             "type": "string",
-            "example": "active"
+            "example": "My Flow"
           },
-          "current_status": {
-            "type": "string",
-            "example": "active"
+          "graph": {
+            "$ref": "#/components/schemas/FlowGraph"
           },
-          "default_mapper_type": {
+          "type": {
             "type": "string",
-            "example": "jsonata"
+            "description": "Flow type",
+            "enum": [
+              "ordinary",
+              "realtime"
+            ]
           },
-          "description": {
-            "type": "string",
-            "example": "A description"
-          }
-        }
-      },
-      "TokenRequest": {
-        "type": "object",
-        "required": [
-          "username",
-          "password"
-        ],
-        "properties": {
-          "username": {
-            "type": "string",
-            "example": "user@example.com"
+          "userId": {
+            "type": "string"
           },
-          "password": {
-            "type": "string",
-            "example": "12345abc"
-          }
-        }
-      },
-      "TokenResponse": {
-        "type": "object",
-        "properties": {
-          "token": {
-            "type": "string",
-            "example": "123abc.456def.789ghi"
+          "workspaceId": {
+            "type": "string"
           }
```

Comment:
Newest version has some adjustments here. Instead of `workspaceId` an owner array is used. See: https://app.swaggerhub.com/apis/drobiazko/flows-repository/0.0.5#/

Reply:
Done